### DISCUSSION TITLE
[Avoid Yourself Challenge] Exceeding 100 turns and running itself without other options results in lose

### DISCUSSION THREAD

**Discussion by coreyja** [BattlesnakeOfficial/feedback#49] [Bug Reports]
My snake exceeds the 100 turns, and eventually eats a lot of food and runs out of options, and runs into himself.

This seems like its enough to trigger a lose 😢 

That _feels_ like it fulfills the spirit of the challenge so is maybe a bug?

**Comment by andrewdmaclean** [comment]
Hey @coreyja 👋 Sorry for the delay. Do you have a link to the game?

**Comment by coreyja** [comment]
Here is an example (just ran it): https://play.battlesnake.com/g/8c257114-daf6-454f-a3bb-dc4e8a67693b/?title=Avoid%20Yourself

At turn `236` my snake has boxed himself into a corner. Since he doesn't consider moving off the board (ever) he runs into himself, but it is AFTER the 100 turns.

**Comment by chris-bsnake** [comment]
The challenge is working as intended. Generally, with challenges we are expecting developers to tweak their snakes for each type of challenge. In this case the challenge has a hard rule that you can't run into yourself.

**Reply by coreyja** [reply]
That makes sense!

The hover tooltip for the challenge DOES indicate that as well: It currently reads `Survive for 100 turns and don't hit yourself`

**Reply by chris-bsnake** [reply]
Yes, we want to do another iteration on the layout of that page to make the instructions much easier to find.  Hiding them in a tooltip is not great.
